Tools/BasicStation-Certificates-Generation/helper-functions

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```

Comment:
This file has CRLF line ending when checking out on windows (the otherone being LF)

Reply:
Changed to LF, thanks

Reply:
@danigian. I'm curious how you changed the line-endings because I don't see anything changed in `.gitattributes`. Normally, all `*.sh` files are configured for LF but since these are without any extension, you'll need to explicitly list this one.

Note that Git says `certificate-generate.sh` is using `LF` but the second one is still `CRLF`:

    > git ls-files --eol Tools/BasicStation-Certificates-Generation/
    i/lf    w/lf    attr/text eol=lf        Tools/BasicStation-Certificates-Generation/certificate-generate.sh
    i/lf    w/crlf  attr/text=auto          Tools/BasicStation-Certificates-Generation/helper-functions

Reply:
@atifaziz, I actually did the change there, but forgot to push 😩 
Can you please check now?

Reply:
> git ls-files --eol Tools/BasicStation-Certificates-Generation/
    i/lf    w/lf    attr/text eol=lf        Tools/BasicStation-Certificates-Generation/certificate-generate.sh
    i/lf    w/lf    attr/text eol=lf        Tools/BasicStation-Certificates-Generation/helper-functions

👍🏼